src/carebot-tracker.js

```diff
@@ -304,23 +313,35 @@
             var percent = depthPercent();
             var bucket = percentBucket(percent);
             if (bucket > previousBucket) {
-                var seconds = timer.check().seconds;
-                callback(bucket, seconds);
-            } else {
-                // The user is scrolling back up.
+                callback(bucket, timer.check().seconds);
                 previousBucket = bucket;
```

Comment:
For sure. We want to cover the situation where someone skims or scrolls down quickly (eg to see how long the article is), and then re-reads. 

I've tried to keep the tracking code analytics-agnostic so people can use the stats in different ways, and don't necessarily have to re-report them to GA. For example, we're thinking about testing out a different system for tracking this event data. 

As far as the 500 event limit goes, for most articles I think we're only seeing 10-15 events from the scroll depth tracker. The bigger one is the linger time indicator. I need to add code to stop reporting once the page is inactive for a certain amount of time.